tests/AulaTests/Stories/Tests.hs

```diff
@@ -5,7 +5,7 @@
 {-# LANGUAGE ScopedTypeVariables   #-}
 {-# LANGUAGE TypeOperators         #-}
 
-{-# OPTIONS_GHC -Wall -Werror #-}
+{-# OPTIONS_GHC -Wall -Werror -fno-warn-missing-signatures #-}
```

Comment:
I like the missing-signatures warning and am skeptical about disabeling it, but I seem to be in the minority, so I'll leave it as is.